auxx/src/Plugin.hs

```diff
@@ -77,14 +76,8 @@ rawExec ::
 rawExec mHasAuxxMode AuxxOptions{..} mSendActions = \case
     Left WithCommandAction{..} -> do
         printAction <- getPrintAction
-        let commandProcs = createCommandProcs mHasAuxxMode printAction mSendActions
         printAction "... the auxx plugin is ready"
-        forever $ withCommand $ \line -> case Lang.parse line of
-            Left parseError -> printAction (show . Lang.ppParseError $ parseError)
-            Right expr -> Lang.evaluate commandProcs expr >>= \eitherValue ->
-                case eitherValue of
-                    Left evalError -> printAction (show . Lang.ppEvalError $ evalError)
-                    Right value -> withValueText printAction value
+        forever $ withCommand $ \line -> runCmd mHasAuxxMode line mSendActions printAction
```

Comment:
You could eta-reduce this if you make `line` the last parameter of `runCmd` (I find it weird that you pass it before configurations)